### PR TITLE
Constrain dependencies in installer's base env more strictly

### PR DESCRIPTION
**Pull request by jaimergp**
Closes https://github.com/napari/napari-plugin-manager/issues/108

### CODE REVIEW

**Review by Czaki — APPROVED** [review]
It means that once conda is updated for new mamba we will need to relax these constraints?

**Comment by jaimergp** [comment]
Good point. Yes, we will have to rebuild. But let me try without the `<2` part, it might work too.

File: build_installers.py [hunk 1]
```diff
@@ -99,6 +99,13 @@
 else:
     raise RuntimeError(f"Unrecognized OS: {sys.platform}")
 
+CONDA_TOOL_DEPS = (
+    "conda >=23.10",
+    "conda-libmamba-solver",
+    "mamba <2.0a0",
```
Comment (jaimergp) [inline build_installers.py:105]:
```suggestion
    "mamba",
```

**Comment by jaimergp** [comment]
Yea, a simple, unconstrained `mamba` works too. Let's merge this and fix rc3.

**Pull request merged**